cli/tools/test_runner.rs

```diff
@@ -540,15 +541,18 @@ pub async fn run_tests(
 
             if let TestResult::Failed(_) = result {
               has_error = true;
+              failed += 1;
             }
           }
           _ => {}
         }
 
         reporter.visit_event(event);
 
-        if has_error && fail_fast {
-          break;
+        if let Some(x) = fail_fast {
+          if failed >= x {
```

Comment:
Actually, this fails on the Nth failure not after N failures as per the help text.

Reply:
so, we change the help text by 
```
Stop after Nth failure
```

Reply:
`Stop after N errors` would be best

Reply:
Yes , I share the same opinion 👍🏼